src/frontend/src/pages/Index/Settings/AdminCenter/Index.tsx

```diff
@@ -61,10 +65,6 @@ const CurrencyTable = Loadable(
   lazy(() => import('../../../../tables/settings/CurrencyTable'))
 );
 
-const TemplateManagementPanel = Loadable(
-  lazy(() => import('./TemplateManagementPanel'))
-);
```

Comment:
Why have you removed the lazy loading here? This means that now when someone opens the admin center the complete huge chunk for the code editor is loaded too.

Reply:
This is because we now use two instances of the "template table" in the panel - and (as far as I can tell) you cannot use lazy importer for components you are passing props to? At least it does not seem to work that way...

Reply:
I would suggest moving them both into one separate file and have this index route just for linking all sub-pages of the admin center together instead of defining some page logic in the Index.tsx of the admin center.

But why do you mean lazy should not work with props? That should work fine, we would just need to use generics [here](https://github.com/inventree/InvenTree/blob/afc66d4cc9a868e3e080c968fda525f383fe3b63/src/frontend/src/functions/loading.tsx#L19-L24) so that the props will be properly introspected for TS. But if you ignore the TS error, that lazy loading should work also with props.